src/components/Stock.js

```diff
@@ -1,15 +1,22 @@
-import React from "react";
+import React from "react"
+
+function Stock({ stock, handleBuy, handleDelete }) {
+  const { ticker, name, price } = stock
 
-function Stock() {
   return (
     <div>
-      <div className="card">
+      <div
+        onClick={() => (handleBuy ? handleBuy(stock) : handleDelete(stock))}
```

Comment:
This could turn into: 
```suggestion
        onClick={() => handleClick(stock)}
```